src/style.css

```
@@ -1 +1,88 @@
-/* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+.moyo_img {
+  width: 40px;
+  height: 40px;
+  left: 50px;
+  top: 20px;
+  margin-top: 10px;
+}
+
+.moyo_icon {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  padding: 0;
+  display: flex;
+  align-items: center;
+}
+
+.moyo_header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+/* .nav {
+  display: flex;
+
+} */
+
+.nav_link {
+  position: relative;
+  display: block;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  padding: 23px 0;
+}
+
+.nav_link:last-child {
+  margin-right: 0;
+}
+
+.nav_item {
+  display: block;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  margin-right: 20px;
+  list-style: none;
+}
+
+.nav_item:last-child {
+  margin-right: 0;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  width: 37px;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+  bottom: 0;
+  right: 0;
+  left: 0;
```

```suggestion
  right: 0;
  left: 0;
```
Why do you need both?